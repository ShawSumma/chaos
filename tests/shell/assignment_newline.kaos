num b = 123
b = 
