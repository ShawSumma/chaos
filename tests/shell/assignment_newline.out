[1;46m  Syntax error:
Module: ~/chaos/__interactive__.kaos
[0;46m    Line: 2
[0;46m    Cause: \n

You have chosen the order!
